emr/instant.json

```
@@ -0,0 +1,8 @@
+{
+  "id": "emr",
+  "name": "emr",
+  "description": "",
+  "type": "infrastructure",
+  "version": "v0.0.1",
+  "dependencies": ["core"]
+}
```

Please add newline to end of file for consistency and because some interpreters require this

```suggestion
}

```

---

Thanks , done